torchinductor/lowering.py

```diff
@@ -2173,12 +2173,14 @@ def max_pool2d_with_indices_backward(
             h // stride[0] - max(0, (h - kernel_size[0]) // stride[0])
             for h in range(kernel_size[0] * 2)
         ]
+        + [1]
```

Comment:
nit, i think it would be a little more intuitive to fold the one into the expression than as `+ [1]`